### PR TITLE
Add scritps to launch/stop MIX servers

### PR DESCRIPTION
**Pull request by maropu**
This pr adds scripts to easily launch and stop MIX servers in a group of machines;
we add a list of hosts in `conf/mixserv_list`, then invoke `./bin/run_mixserv_cluster.sh` to launch MIX servers in the hosts. `./bin/stop_mixserv_cluster.sh` shuts down the servers.
We also query the running statuses of the servers by using  `./bin/status_mixserv_cluster.sh`.

### CODE REVIEW

**Comment by myui** [comment]
Thanks. This is very useful.
Then, how about making it to an init.d script?

```sh
/sbin/chkconfig mixserv on
/sbin/service mixserv start|restart|stop|status
```

Here is my init.d script for hadoop datanode.
https://gist.github.com/myui/ba026db1b21ce834382e

[1] http://refspecs.linuxbase.org/LSB_4.1.0/LSB-Core-generic/LSB-Core-generic/tocsysinit.html
[2] http://qiita.com/ryo0301/items/0bc0f1db423b2da352fe (in Japanese)

File: bin/mixserv_daemon.sh [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+# Load a version number from a VERSION file
+VERSION=`cat $HIVEMALL_HOME/VERSION`
+
+MIXSERV_PID_DIR="/tmp"
+MIXSERV_PID_FILE="$MIXSERV_PID_DIR/hivemall-mixserv-$VERSION-$USER.pid"
+MIXSERV_LOG_DIR="$HIVEMALL_HOME/logs"
+MIXSERV_LOG_FILE="$MIXSERV_LOG_DIR/hivemall-mixserv-$VERSION-$USER-$HOSTNAME.out"
+
+rotate_mixserv_log() {
```
Comment (myui) [inline bin/mixserv_daemon.sh]: Why not just use RollingFileAppender?
http://2min2code.com/articles/log4j_intro/rolling_file_on_size_prop

Comment (maropu) [inline bin/mixserv_daemon.sh]: ISTM that the logger cannot hold stacktrace messages caused by exceptions. Thought?

**Comment by maropu** [comment]
@myui Good idea though, non-root users cannot use the `init.d` script.
So, we'd better to keep both scripts, `init.d` and `run_mixserv_cluster.sh`.

**Comment by myui** [comment]
@maropu: 

True. Better to have both of them.

**Comment by myui** [comment]
Merged w/ some modifications.
https://github.com/myui/hivemall/pull/241

**Pull request merged**